MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,374 @@
+---
+layout: post
+title: Data annotations in .NET MAUI DataForm control | Syncfusion
+description: Learn about the Data annotations support in Syncfusion Maui DataForm (SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfDataForm
+documentation: UG
+---
+
+# Data annotations in .NET MAUI DataForm (SfDataForm)
+
+The data form supports the following attribute to handle the data, and these attributes can be accessible using `System.ComponentModel.DataAnnotation` assembly.
```

Comment:
The data form supports the following attribute to handle the data, and these attributes can be accessed using the `System.ComponentModel.DataAnnotation` assembly.

Reply:
content updated